python/data_structures/linked_list.py

```diff
@@ -1,16 +1,49 @@
+
 class LinkedList:
     """
     Put docstring here
     """
 
     def __init__(self):
-        # initialization here
-        pass
+        self.head = None
+
+    def insert(self, value):
+        self.head = Node(value, self.head)
+
+    def __str__(self):
+        null = 'NULL'
+        if self.head is None:
+            return null
```

Comment:
This `if` statement works as intended, but consider how the code would run without lines 15 and 16. Are they necessary?